packages/rn-tester/js/examples/FlatList/FlatListExample.js

```diff
@@ -145,15 +156,30 @@ class FlatListExample extends React.PureComponent<Props, State> {
                   />
                 </View>
               )}
+              {
+               this.state.multiColumn === true && (
+                 <View>
+                   <TextInput
+                      placeholder="Number of columns"
+                      keyboardType={'numeric'}
+                      onChange={(event)=> 
+                        this.setState({
+                          numColumns: Number(event.nativeEvent.text)
+                        })
+                      }
+                    />
+                </View>
+               )
+              }
               <Spindicator value={this._scrollPos} />
             </View>
           </View>
           <SeparatorComponent />
           <Animated.FlatList
             fadingEdgeLength={this.state.fadingEdgeLength}
             ItemSeparatorComponent={ItemSeparatorComponent}
-            ListHeaderComponent={<HeaderComponent />}
-            ListFooterComponent={FooterComponent}
+            ListHeaderComponent={this.state.header ? <HeaderComponent />: null}
+            ListFooterComponent={this.state.footer ? FooterComponent : null}
```

Comment:
@stealthanthrax I think we should use either angle brackets notation or simple variable notation, not both of them for better consistency.

Either: <HeaderComponent /> and <FooterComponent/>
or
HeaderComponent and FooterComponent

Both works!